include/prometheus/exposer.h

```diff
@@ -18,7 +18,10 @@ class MetricsHandler;
 
 class Exposer {
  public:
-  explicit Exposer(const std::string& bind_address,
+  using CollectableContainer = std::vector<std::weak_ptr<Collectable>>;
+
+  explicit Exposer(CollectableContainer&& collectables,
```

Comment:
The code style in the rest of the code base so far is having references attached to types, rather than variable names @jovany-wang . Why do you suggest changing it?

Reply:
Oops, sorry. That's my personal prefer. 
It's fine to follow the code style of  the codebase on master branch.